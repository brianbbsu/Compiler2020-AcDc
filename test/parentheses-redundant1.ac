i a
f b
i c
i d
i e

a = (((((18)))))
p a

b = (((0.5)))
p b

c = (((2 + 3)))
d = (((((c)))))
e = (((c + (2))) - ((((d) - 1))))
p c
p d
p e
