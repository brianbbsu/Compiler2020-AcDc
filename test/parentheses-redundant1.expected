18
0.50000
5
5
3
